bin/lib/create.js

```diff
@@ -125,16 +131,24 @@ function writeProjectProperties(projectPath, target_api) {
     fs.writeFileSync(dstPath, data);
 }
 
-function prepBuildFiles(projectPath) {
+// This makes no sense, what if you're building with a different build system?
```

Comment:
Would folding this functionality up into the Builder abstraction make sense here? Would different project structures require different building different things at create time as well? Put otherwise: do the Builders have to do specific things at create-time as well?